x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/assignment/AssignmentStats.java

```diff
@@ -462,8 +503,7 @@ public InferenceStats getOverallInferenceStats() {
             nodeStats.stream().filter(n -> n.getInferenceCount().isPresent()).mapToLong(n -> n.getInferenceCount().get()).sum(),
             // This is for ALL failures, so sum the error counts, timeouts, and rejections
             nodeStats.stream().mapToLong(n -> n.getErrorCount() + n.getTimeoutCount() + n.getRejectedExecutionCount()).sum(),
-            // TODO Update when we actually have cache miss/hit values
-            0L,
+            nodeStats.stream().filter(n -> n.getCacheHitCount().isPresent()).mapToLong(n -> n.getCacheHitCount().get()).sum(),
```

Comment:
`InferenceStats` expects here the cache **miss** count, right? We shouldn't be passing the hit count.

Reply:
Yes, good point.  That relates to the clashing field name conversation we had earlier.  I thought I'd reverted that line but obviously not - lucky you spotted it.